dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -452,13 +459,34 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 								}
 							/>
 						</Island>
+						{isMainMediaTest && (
+							<Hide from="desktop">
+								<MainMedia
+									format={format}
+									elements={CAPIArticle.mainMediaElements}
+									adTargeting={adTargeting}
+									host={host}
+									pageId={CAPIArticle.pageId}
+									webTitle={CAPIArticle.webTitle}
+									ajaxUrl={CAPIArticle.config.ajaxUrl}
+									switches={CAPIArticle.config.switches}
+									isSensitive={CAPIArticle.config.isSensitive}
+									isAdFreeUser={CAPIArticle.isAdFreeUser}
+									isMainMediaTest={isMainMediaTest}
+								/>
+							</Hide>
+						)}
 					</Section>
 				) : (
 					<Section
 						fullWidth={true}
 						showTopBorder={false}
 						backgroundColour={palette.background.header}
+						innerBackgroundColour={palette.background.header}
+						showSideBorders={isMainMediaTest ? false : true}
 						borderColour={palette.border.headline}
+						format={format}
+						isMainMediaTest={isMainMediaTest}
```

Comment:
I can't see this border vary in the screenshots, can you elaborate more on why the test is affecting this?

Reply:
Sure! The borders are shown by default. Previously we've shown a vertical bleed next to the image on tablet which bleeds into the borders. The new designs remove this bleed but without removing the borders we have a thin pillar line around the image. Here's some pictures to show more clearly...

`showSideBorders={true}` - default
![Screenshot 2022-09-20 at 09 21 27](https://user-images.githubusercontent.com/20416599/191206426-292b1c55-28f8-436c-b61e-94881c097e96.png)

`showSideBorders={false}`
![Screenshot 2022-09-20 at 09 20 56](https://user-images.githubusercontent.com/20416599/191206451-1403395e-23e4-43af-a094-746acca5e7d9.png)